src/components/DisplayNames/DisplayNamesWithTooltip.js

```diff
@@ -0,0 +1,92 @@
+import React, {Fragment, useCallback, useEffect, useRef, useState} from 'react';
+import {View} from 'react-native';
+import _ from 'underscore';
+import styles from '../../styles/styles';
+import Text from '../Text';
+import Tooltip from '../Tooltip';
+import DisplayNamesTooltipItem from './DisplayNamesTooltipItem';
+import {defaultProps, propTypes} from './displayNamesPropTypes';
+
+function DisplayNamesWithToolTip(props) {
+    const containerRef = useRef(null);
+    const childRefs = useRef([]);
+    const [isEllipsisActive, setIsEllipsisActive] = useState(false);
+
+    useEffect(() => {
+        setIsEllipsisActive(
+            containerRef.current && containerRef.current.offsetWidth && containerRef.current.scrollWidth && containerRef.current.offsetWidth < containerRef.current.scrollWidth,
+        );
+    }, []);
```

Comment:
Ellipsis is not updated when the `fullTitle` prop is changed which is lead to this issue - https://github.com/Expensify/App/issues/27545